infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -126,26 +128,21 @@
      */
     public SequencerServer(ServerContext serverContext) {
         this.serverContext = serverContext;
-        Map<String, Object> opts = serverContext.getServerConfig();
+        Config config = Config.parse(serverContext.getServerConfig());
+
         this.executor = Executors.newFixedThreadPool(
                 serverContext.getSequencerThreadCount(),
                 new ServerThreadFactory("sequencer-", new ServerThreadFactory.ExceptionHandler())
         );
 
-        long initialToken = Utils.parseLong(opts.get("--initial-token"));
+        long initialToken = config.getInitialToken();
```

Comment:
Is there any point of letting non address being set in config builder? if not why not moving this logic to config builder?

Reply:
fixed